x-pack/plugins/security_solution/public/timelines/components/flyout/button/index.test.tsx

```diff
@@ -40,7 +47,13 @@ describe('FlyoutButton', () => {
 
     const wrapper = mount(
       <TestProviders>
-        <FlyoutButton dataProviders={[]} onOpen={onOpen} show={true} timelineId="test" />
+        <FlyoutButton
+          dataProviders={[]}
+          onOpen={onOpen}
+          show={true}
+          timelineId="test"
+          timelineType={TimelineType.default}
```

Comment:
do we really need to pass a new Props of `timelineType`, can we just default it?